src/components/create.rs

```diff
@@ -30,14 +42,36 @@ pub(crate) fn Create() -> Element {
     let npub_arbitrator = use_signal(String::new);
     let amount_buyer = use_signal(String::new);
     let amount_seller = use_signal(String::new);
-    let fee_rate = use_signal(|| "1".to_string());
+    let mut fee_rate = use_signal(String::new);
+    let fee_estimates = use_signal(|| Option::<HashMap<u16, f64>>::None);
     let timelock_days = use_signal(String::new);
     let timelock_hours = use_signal(String::new);
     let funding_txid = use_signal(String::new);
     let mut escrow_address_str = use_signal(String::new);
     let mut escrow_transaction = use_signal(String::new);
     let mut derived_address_buyer = use_signal(String::new);
     let mut derived_address_seller = use_signal(String::new);
+
+    use_effect(move || {
+        to_owned![fee_estimates];
+
+        spawn(async move {
+            match fetch_fee_estimates().await {
+                Ok(estimates) => {
+                    #[cfg(debug_assertions)]
+                    trace!("Fee estimates fetched successfully: {:?}", estimates);
```

Comment:
We are using this convention in the codebase:
```suggestion
                    trace!(?estimates, "Fee estimates fetched successfully");
```

Reply:
Done